mailer/Dockerfile

```
@@ -1,9 +1,20 @@
-FROM python:3.8-slim-buster
+ARG PYTHONIMAGE=python:3.9.3-slim
+FROM $PYTHONIMAGE AS prepare
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-RUN pip install poetry && \
-    mkdir -p /opt
-COPY mailer.py poetry.lock pyproject.toml start.sh  /opt/
-WORKDIR /opt
-RUN poetry install --no-dev
+RUN pip install --no-cache-dir poetry
 
-CMD ["/opt/start.sh"]
+WORKDIR /app
+COPY poetry.lock /app
+COPY pyproject.toml /app
+RUN poetry config virtualenvs.in-project true \
+	&& poetry install --no-dev
+
+FROM $PYTHONIMAGE
```

```suggestion
FROM python:3.9.6-slim
```
Ved å bruke $PYTHONIMAGE, klarer ikke Dependabot å sjekke versjon som brukes. Og Dependabot vil oppdatere begge i en eventuell PR